ci-operator/step-registry/upi/conf/vsphere/upi-conf-vsphere-commands.sh

```diff
@@ -51,7 +51,6 @@ else
   if ! grep -qF 'unknown command "coreos"' /tmp/err.txt; then
     echo "Unhandled error from openshift-install" 1>&2
     cat /tmp/err.txt
-    exit 1
```

Comment:
If `coreos` subcommand doens't exist should we just immediately go to legacy installer json?

Removing `exit 1` doesn't make sense unless we just want reporting of errors.

Reply:
Hmm the intention here was that we'd only do this if we got some unexpected error.  But...this must not be working for some reason.

Reply:
Oh...oh wow:  it's because the installer's logging library does something different when it's outputting to a tty vs not. 

```
$ openshift-install-4.7 coreos print-stream-json
FATA[0000] Error executing openshift-install: unknown command "coreos" for "openshift-install-4.7" 
$ openshift-install-4.7 coreos print-stream-json 2>/tmp/err.txt
$ cat /tmp/err.txt 
time="2021-04-01T18:01:26Z" level=fatal msg="Error executing openshift-install: unknown command \"coreos\" for \"openshift-install-4.7\""
$
```

Which...OK I understand.  So I think we could also just fix the quotes here in the grep.  But if you want to just remove the error message check, OK by me too.

Reply:
@cgwalters can you see if this update makes sense?
I was testing on the cli and the `-F` in the grep seems to cause the issue